### PR TITLE
A couple of ideas for meetings

### PR DESCRIPTION
**Pull request by markwilkie**
Let me know what y'all think.

### CODE REVIEW

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 1]
```diff
@@ -7,10 +7,14 @@ In this new hybrid world there are many challenges ahead.  This is our current a
 - All of our meetings must be inclusive of everyone who wants to participate.
 - Meetings should be reserved for situations and discussions that are unreasonable or silly to have async.
 - Meetings are an important, necessary, and key part of belonging.
+- Social interactions are also a priority.  (not just the agenda)
+- Nobody should feel like they *only* go to meetings all day
```
Comment (jonfortescue) [inline Documentation/TeamProcess/DevGuide/meetings.md:11]: rip managers

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 2]
```diff
@@ -7,10 +7,14 @@ In this new hybrid world there are many challenges ahead.  This is our current a
...
 
 ## Guidelines
 - Every meeting is a Teams meeting.  (regardless if synchronous or async)
+- Meetings should start 10 minutes after the hour (or half hour) so that folks have breathing space in between.
```
Comment (jonfortescue) [inline Documentation/TeamProcess/DevGuide/meetings.md:15]: honestly I'd prefer meetings ending 10 minutes before the hour, personally. I know that's like super nitty and might just be me but I like round numbers for start times

Comment (markwilkie) [reply Documentation/TeamProcess/DevGuide/meetings.md:15]: ending 10 minutes before is harder to actually adhere to than simply starting late is what I've seen...

File: Documentation/TeamProcess/DevGuide/meetings.md (hunk 1 above)
Comment (adiaaida) [reply Documentation/TeamProcess/DevGuide/meetings.md:11]: RIP my thursdays

File: Documentation/TeamProcess/DevGuide/meetings.md [hunk 3]
```diff
@@ -7,10 +7,14 @@ In this new hybrid world there are many challenges ahead.  This is our current a
...
 - Every meeting has recording turned on by default
+- A meeting agenda articulating what is hoped to be accomplished is highly encouraged.
```
Comment (premun) [inline Documentation/TeamProcess/DevGuide/meetings.md:17]: +1 this is important in my opinion to make the meetings much more effective

**Review by premun — APPROVED** [review]

**Review by garath — APPROVED** [review]

**Pull request merged**